system_tests/test_cli_package.py

```diff
@@ -241,54 +241,41 @@ def bootstrap_manager(self):
         if self.is_install_plugins():
             install_plugins = '--install-plugins'
         out = self._execute_command(
-            'bootstrap -p {0} -i {1} {2}'.format(
-                self.test_manager_blueprint_path,
-                self.remote_bootstrap_inputs_path,
+            """bootstrap -p {0} -i "{1}" {2}""".format(
+                self.manager_blueprint_path,
+                json.dumps(inputs).replace('"', "'"),
```

Comment:
AFAIK the json.dumps string is different from our required inputs format of "key1=value1;key2=value2"